components/sendgrid/sendgrid.app.js

```diff
@@ -118,5 +113,429 @@ module.exports = {
     async disableSignedWebhook() {
       return this._setSignedWebhook(false);
     },
+    /**
+     * Adds an email to global supressions
+     *
+     * @param {array}  recipientEmails the email addresses to be added to the global
+     * suppressions group.
+     * @returns {recipient_emails: array} the email addresses that are globally suppressed.
+     */
+    async addEmailToGlobalSupression(recipientEmails) {
+      const config = {
+        method: "POST",
+        url: "/v3/asm/suppressions/global",
```

Comment:
If you create a function such as `_makeRequest(customConfig)` you would just need to pass parameters like this:
```javascript
const customConfig = {
  method: "post",
  path: "/asm/suppressions/global",
  ...args,
};
await this._makeApiRequest(customConfig);
```

Reply:
I just addressed this comment.
but named it `_makeClientRequest`.